src/PoolManager.sol

```diff
@@ -127,16 +127,16 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
     }
 
     /// @inheritdoc IPoolManager
-    function lock(bytes calldata data) external payable override returns (bytes memory result) {
-        if (Lock.isLocked()) revert AlreadyLocked();
+    function unlock(bytes calldata data) external payable override returns (bytes memory result) {
+        if (Lock.isUnlocked()) revert AlreadyUnlocked();
 
-        Lock.lock();
+        Lock.unlock();
 
         // the caller does everything in this callback, including paying what they owe via calls to settle
-        result = ILockCallback(msg.sender).lockAcquired(data);
+        result = IUnlockCallback(msg.sender).managerUnlocked(data);
```

Comment:
mm `managerUnlocked` is the only one im not sure about
maybe it should just be `IUnlockCallback.unlockCallback` or something
lets see what @snreynolds thinks

Reply:
I think`IUnlockCallback.unlockCallback` is fine!